### PR TITLE
Change the Replit tutorial since it was too outdated

### PR DESCRIPTION
**Pull request by cupglassDEV**
idk what to say here, just look at `src/docs/replit/index.md`

### CODE REVIEW

File: src/docs/replit/index.md [hunk 1]
```diff
@@ -21,10 +23,15 @@ Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with
         "twitter": "<twitter-username>"
     },
     "record": {
-        "CNAME": "<siteid>.id.repl.co"
+        "A": "<IP here, listed on the A Record>",
+        "TXT": "<TXT value here, listed on the TXT Record>"
```
Comment (CuteDog5695) [inline src/docs/replit/index.md:27]: A records need to be in an array
```suggestion
        "A": ["<IP here, listed on the A Record>"],
        "TXT": "<TXT value here, listed on the TXT Record>"

```

**Comment by cupglassDEV** [comment]
oh i forgot it, srry

File: src/docs/replit/index.md [hunk 2]
```diff
@@ -3,16 +3,18 @@
...
 ## Connect your repl to your is-a.dev subdomain
-Follow the instructions in the [Replit Custom Domains Guide](https://docs.replit.com/hosting/custom-domains#connecting-your-domain-to-your-repl).
-
-Only follow the "Connecting your domain to your repl" section, then return to this guide for the next steps.
+Since Replit Workspaces has changed alot, follow the instructions in the [Replit Deployment Custom Domains Guide](https://docs.replit.com/cloud-services/deployments/custom-domains). You may need the Replit Deployment subscription to do so
 
 ## Creating the domain file
-Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content:
+After clicking "Link a Domain" button on your Replit Deployment menu and already typed the domain, you will see this
+![image](https://github.com/user-attachments/assets/32abb589-5fad-4383-8554-eebe005052e9)
```
Comment (CuteDog5695) [inline src/docs/replit/index.md:12]: Please upload the image locally

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]

File: src/docs/replit/index.md (hunk 2 above)
Comment (cupglassDEV) [reply src/docs/replit/index.md:12]: On the same `replit` folder?

Comment (CuteDog5695) [reply src/docs/replit/index.md:12]: No, you upload the image at the `img` directory and then change this file to point to that image

Comment (cupglassDEV) [reply src/docs/replit/index.md:12]: Oh sh*t, it turns out on the `img` folder, i will resolve this

**Comment by cupglassDEV** [comment]
I will make a new pr for this, missing one file

**Comment by cupglassDEV** [comment]
https://github.com/is-a-dev/docs/pull/41

**Pull request closed without merging**